### DISCUSSION TITLE
Notifica cambio password

### DISCUSSION THREAD

**Discussion by maelaborazioni** [globaleaks/globaleaks-whistleblowing-software#3721] [Ideas]
Buongiorno,
compilavo una checklist giratami dal nostro avvocato per la stesura della privacy policy.
Tra le varie domande relative alla sicurezza viene chiesto _L'utente viene avvisato del cambio della password?_
Dai miei test sembrerebbe di no (potrei sbagliarmi), potrebbe essere una implementazione sensata?

**Comment by evilaliv3** [comment]
Può spiegare meglio la domanda?
Quale utente dovrebbe essere avvisato del cambio password e in quali situazioni?

**Comment by maelaborazioni** [comment]
L'admin del sito ed i manager (ovvero quelli in possesso di credenziali visto che i segnalanti non hanno accesso con credenziali).
Credo che la feature serva a garantire la sicurezza circa l'accesso con il proprio account.
In realtà facendo alcune prove ho notato che, se qualcuno desidera cambiare la propria password, per farlo deve fornire la propria chiave di recupero. Questo credo che sia una misura di sicurezza in quest'ottica ed anche più sicura se vogliamo poichè agisce a monte e non a valle quando ormai è troppo tardi, è corretto?
